### PR TITLE
APIML Client certificate authentication

### PR DESCRIPTION
**Pull request by jandadav**
🚨🚨🚨 dont merge for 1.16 🚨🚨🚨

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/api-mediation/api-gateway-configuration.md [hunk 1]
```diff
@@ -158,90 +51,83 @@ To change this default configuration, include the following parameters:
...
+3. Restart Zowe&trade.
 
-**Note:** For more information about Hystrix configuration parameters, see the [Netflix - Hystrix documentation](https://github.com/Netflix/Hystrix/wiki/Configuration#execution.isolation.strategy).
+## Gateway timeouts
 
-## Additional parameters
+* **apiml.gateway.timeoutMillis**
 
-* **ibm.serversocket.recover**
+    This property is used to define global value for http/ws client timeout. For finer control, you can override the properties below.
```
Comment (janan07) [inline docs/user-guide/api-mediation/api-gateway-configuration.md:87]: Define 'finer control'?

Comment (jandadav) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: instead of global timeout, a timeout can be set for each individual component

Comment (jandadav) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: 1 timeout value for everything, or many timeout values for different things

**Comment by jandadav** [comment]
@janan07 Hi Andrew, I did review the branch as there has been content pushed to staging that invalidated some of the content. Please feel free to review so we can merge this branch. It has been quite some merge to get it done so I'd like to get it in as soon as possible. Your review is greatly appreciated.

File: docs/user-guide/api-mediation/api-gateway-configuration.md [hunk 2]
```diff
@@ -158,90 +51,83 @@ To change this default configuration, include the following parameters:
...
-                isolation:
-                    thread:
-                        timeoutInMilliseconds: ${apiml.gateway.timeoutMillis}
-                    strategy: SEMAPHORE
-                    semaphore:
-                        maxConcurrentRequests: 100000
-```
+    When z/OSMF is used as an authentication provider, to allow client certificate authentication it is necessary to provide
+valid APPLID. The API ML authenticates to the z/OSMF via generation of the passticket and using it to authenticate to the
+z/OSMF. The value in the default installation of the z/OSMF is `IZUDFLT`. To change the value to another one use the following procedure.
```
[the remark this answers is not in the document]
Comment (jandadav) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: There has been updates. Please check the new doc

File: docs/user-guide/api-mediation/api-gateway-configuration.md (hunk 1 above)
Comment (jandadav) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: There has been updates. Please check the new doc

File: docs/user-guide/api-mediation/api-gateway-configuration.md [hunk 3]
```diff
@@ -1,33 +1,51 @@
 # Advanced Gateway features configuration
 
-As a system programmer who wants to configure advanced Gateway features of the API Mediation Layer, set the following parameters by modifying the `<Zowe install directory>/components/api-mediation/bin/start.sh` file. The parameters begin with the `-D` prefix, similar to all the other parameters in the file. Zowe needs to be restarted for these changes to take effect.
+As a system programmer who wants to configure advanced Gateway features of the API Mediation Layer, set the following parameters by modifying either:
+- `<Zowe install directory>/components/api-mediation/bin/start.sh` file. The parameters begin with the `-D` prefix, similar to all the other parameters in the file. Zowe needs to be restarted for these changes to take effect.
```
Comment (janan07) [inline docs/user-guide/api-mediation/api-gateway-configuration.md:4]: I think something is missing here. You say, "by modifying either", but then there is only one shell.

Comment (janan07) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: Please check this meta-description for accuracy.

**Review by janan07 — APPROVED** [review]
Reviewed and editing by Tech Info. Please review comments prior to merging.

File: docs/user-guide/api-mediation/api-gateway-configuration.md (hunk 3 above)
Comment (jandadav) [reply docs/user-guide/api-mediation/api-gateway-configuration.md]: Thanks for the catch! the current file lists two items: 
- `<Zowe install directory>/components/api-mediation/bin/start.sh` 
- `<Zowe instance directory>/instance.env`

**Pull request merged**